Sources/MongoDBVapor/MongoDBVapor.swift

```diff
@@ -20,9 +20,17 @@ extension Application {
          * Within `Request` handlers, it is preferable to use `Request.mongoDB.client` as that will return a client
          * which uses the same `EventLoop` as the `Request`.
          */
-        public private(set) var client: MongoClient {
+        public var client: MongoClient {
```

Comment:
I realized a user could run into an issue if they do something like this (which we are doing in one of our tests, minus the invalid connection string):
```swift
let app = Application()
defer {
    app.mongoDB.cleanup()
    app.shutdown()
}

try app.configureMongoDB("some invalid connection string")
```

In that case. we'd unconditionally execute the cleanup function from the defer block and crash because `Application.mongoDB.client` was never set.

This change allows us to just no-op on cleanup if the client didn't get properly initialized.